api/router.go

```diff
@@ -88,6 +88,8 @@ func (r *router) forceAddRouter(base, endpoint string, handler http.Handler) err
 	endpoints[endpoint] = handler
 	r.routes[base] = endpoints
 	r.router.Handle(url, handler)
+	url = "/api" + endpoint
```

Comment:
I think you'll notice we're not prescriptive and don't hard code this kinda stuff. Please find a way not to hard code "api" so that each VM can choose what their rest endpoint will be.

Reply:
I have messaged you on discord.